### DISCUSSION TITLE
load url image api wrong angular 14

### DISCUSSION THREAD

**Discussion by germanbeta** [naver/egjs-flicking#775] [Q&A]
When I load the slider with urls loaded from an api, the images are loaded wrong or incomplete and I need to manually resize the window to fix it, but when I place the url directly in the slider it works correctly. Can someone give me an idea of ​​what I'm doing wrong?

**Comment by malangfox** [comment]
Hello @germanbeta.

The response that contains urls from your API may not be immediate. It will take some time for the backend to respond to your request.
But when you put the url directly in the panel, it won't take any time to update the panel size after rendered once.

There seems to be a difference between these two that determines whether or not to use `resize`.
